README.md

```diff
@@ -209,208 +211,4 @@ use and development:
- * Juan Pablo Fernandez
- * Christophe Fillot
- * Mike Frysinger
- * Remi Gacogne
- * Artem Germanov
- * Alexander von Gernler
- * Diego Giagio
- * Artur Grabowski
- * Diwaker Gupta
- * Kuldeep Gupta
- * Sebastian Hahn
- * Dave Hart
- * Greg Hazel
- * Nicholas Heath
- * Michael Herf
- * Savg He
- * Mark Heily
- * Maxime Henrion
- * Michael Herf
- * Greg Hewgill
- * Andrew Hochhaus
- * Aaron Hopkins
- * Tani Hosokawa
- * Jamie Iles
- * Xiuqiang Jiang
- * Claudio Jeker
- * Evan Jones
- * Marcin Juszkiewicz
- * George Kadianakis
- * Makoto Kato
- * Phua Keat
- * Azat Khuzhin
- * Alexander Klauer
- * Kevin Ko
- * Brian Koehmstedt
- * Marko Kreen
- * Ondřej Kuzník
- * Valery Kyholodov
- * Ross Lagerwall
- * Scott Lamb
- * Christopher Layne
- * Adam Langley
- * Graham Leggett
- * Volker Lendecke
- * Philip Lewis
- * Zhou Li
- * David Libenzi
- * Yan Lin
- * Moshe Litvin
- * Simon Liu
- * Mitchell Livingston
- * Hagne Mahre
- * Lubomir Marinov
- * Abilio Marques
- * Nicolas Martyanoff
- * Abel Mathew
- * Nick Mathewson
- * James Mansion
- * Nicholas Marriott
- * Andrey Matveev
- * Caitlin Mercer
- * Dagobert Michelsen
- * Andrea Montefusco
- * Mansour Moufid
- * Mina Naguib
- * Felix Nawothnig
- * Trond Norbye
- * Linus Nordberg
- * Richard Nyberg
- * Jon Oberheide
- * John Ohl
- * Phil Oleson
- * Alexey Ozeritsky
- * Dave Pacheco
- * Derrick Pallas
- * Tassilo von Parseval
- * Catalin Patulea
- * Patrick Pelletier
- * Simon Perreault
- * Dan Petro
- * Pierre Phaneuf
- * Amarin Phaosawasdi
- * Ryan Phillips
- * Dimitre Piskyulev
- * Pavel Plesov
- * Jon Poland
- * Roman Puls
- * Nate R
- * Robert Ransom
- * Balint Reczey
- * Bert JW Regeer
- * Nate Rosenblum
- * Peter Rosin
- * Maseeb Abdul Qadir
- * Wang Qin
- * Alex S
- * Gyepi Sam
- * Hanna Schroeter
- * Ralf Schmitt
- * Mike Smellie
- * Steve Snyder
- * Nir Soffer
- * Dug Song
- * Dongsheng Song
- * Hannes Sowa
- * Joakim Soderberg
- * Joseph Spadavecchia
- * Kevin Springborn
- * Harlan Stenn
- * Andrew Sweeney
- * Ferenc Szalai
- * Brodie Thiesfield
- * Jason Toffaletti
- * Brian Utterback
- * Gisle Vanem
- * Bas Verhoeven
- * Constantine Verutin
- * Colin Watt
- * Zack Weinberg
- * Jardel Weyrich
- * Jay R. Wren
- * Zack Weinberg
- * Mobai Zhang
- * Alejo
- * Alex
- * Taral
- * propanbutan
- * masksqwe
- * mmadia
- * yangacer
- * Andrey Skriabin
- * basavesh.as
- * billsegall
- * Bill Vaughan
- * Christopher Wiley
- * David Paschich
- * Ed Schouten
- * Eduardo Panisset
- * Jan Heylen
- * jer-gentoo
- * Joakim Söderberg
- * kirillDanshin
- * lzmths
- * Marcus Sundberg
- * Mark Mentovai
- * Mattes D
- * Matyas Dolak
- * Neeraj Badlani
- * Nick Mathewson
- * Rainer Keller
- * Seungmo Koo
- * Thomas Bernard
- * Xiao Bao Clark
- * zeliard
- * Zonr Chang
- * Kurt Roeckx
- * Seven
- * Simone Basso
- * Vlad Shcherban
- * Tim Hentenaar
- * Breaker
- * johnsonlee
- * Philip Prindeville
- * Vis Virial
-
-If we have forgotten your name, please contact us.
+fixing bugs: [See CONTRIBUTORS](./CONTRIBUTORS.MD)
```

Comment:
I think that it's better to use absolute path here, since it should mean the same

Reply:
All right, I'll squash all commits and replace that with an absolute path 😄
I'll also start working on the Docs as you had asked